src/content/docs/apis/nerdgraph/examples/nerdgraph-manage-users.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: "NerdGraph tutorial: Manage users"
+tags:
+  - APIs
+  - NerdGraph
+metaDescription: "How to use New Relic's NerdGraph API to manage users and return user information."
+---
+
+You can use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage users. You can edit their user type and email address, and return other types of user information. 
+
+For how to do this in the UI, see the [user management UI docs](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks). 
+
+## Requirements [#requirements]
+
+Some requirements for managing users via NerdGraph: 
+* You must be a user on [our newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models#determine-user-model). Other permissions-related requirements: 
+  * Required [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type): core user or full platform user 
+  * Required [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings): To view and make changes to users requires **Authentication domain settings**. To view and make changes at the organization level requires **Organization settings**. 
+
+## Before you start [#concepts]
+
+Before using NerdGraph to manage users: 
+
+* Ensure you have a decent understanding of our [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#understand-concepts)
+* Note that you can only add users from the UI, not using the API
```

Comment:
In general, I'm not a huge fan of "note" here. I assume if you've put it in a bullet, you want me to notice it :)